src/policies/__id/peoplefinder.DELETE.api.users.__id.rego

```diff
@@ -0,0 +1,25 @@
+package peoplefinder.DELETE.api.users.__id
```

Comment:
renaming all the files is a convention change - I'm not against it per se, but it should be done as a joint decision. adding @gertd for his opinion.